code/datums/martial/dropkicking.dm

```diff
@@ -0,0 +1,81 @@
+/datum/martial_art/dropkick
+	name = "Dropkicking"
+	id = MARTIALART_DROPKICK
+	var/obj/effect/proc_holder/spell/aimed/dropkick/dropkick
+	var/dropkicking = FALSE
+
+/datum/martial_art/dropkick/teach(mob/living/carbon/human/H,make_temporary=0)
+	if(..())
+		to_chat(H, "<span class = 'userdanger'>You know the arts of [name]!</span>")
+		to_chat(H, "<span class = 'danger'>You have gained the powers of DROPKICK! Mouse over the button to see how it works.</span>")
+		var/obj/effect/proc_holder/spell/aimed/dropkick/DK = new
+		H.AddSpell(DK)
+		dropkick = DK
+
+/datum/martial_art/dropkick/on_remove(mob/living/carbon/human/H)
+	to_chat(H, "<span class = 'userdanger'>You suddenly forget the arts of [name]...</span>")
+	if(dropkick)
+		H.RemoveSpell(dropkick)
+
+/datum/martial_art/dropkick/throw_impact_act(atom/hit_atom, datum/thrownthing/throwingdatum)
+	//
+	if(!dropkicking)
+		return//this will call the rest of throw_impact.
+	//owner.Paralyze(20)//you still always take longer to get up because it's a dropkick
+	if(iscarbon(hit_atom) && hit_atom != src)
+		var/mob/living/carbon/victim = hit_atom
+		if(victim.movement_type & FLYING)
+			return FALSE//this does not call the rest of throw_impact.
```

Comment:
Doesn't this fail the if check here, causing the throw_impact to continue?
https://github.com/tgstation/tgstation/blob/cd4b3c4b4fdb0b90eea7f41ae5b4804c08b684ce/code/modules/mob/living/carbon/carbon.dm#L98-L100

Did this work in testing?

Reply:
it absolutely does not i should probably go with
if(!martial_return)
instead so the returns make more sense